### DISCUSSION TITLE
naming convention for roughness length experiments

### DISCUSSION THREAD

**Discussion by apossner** [ARM-Development/comble-mip#42] [General]
Hi all, in the timeline it says to name the experiments "{prefix}_FixN_def_z0", in the overview of the four different experiments, the runs are labeled DHARMA_Lx25km_dx200_z0_1.nc to DHARMA_Lx25km_dx200_z0_4.nc. So would the correct labeling be "{prefix}_FixN_1_z0" or "{prefix}_FixN_z0_1" for the model experiments?

**Comment by twjuliano** [comment]
Hello @apossner,
Thanks for the question and apologies if our instructions were not clear.
We conducted the four z0 sensitivity experiments just to show the impact of the roughness lengths and also to converge on a value for the main experiments that we're requesting from participants.
The requested simulations are listed on [this](https://arm-development.github.io/comble-mip/timeline.html) page in the two Tables for Part I and Part II. Note that for '{prefix}_FixN', we're asking folks use the z0 values set on the [Main Model Configuration page](https://arm-development.github.io/comble-mip/main_configuration.html), and for '{prefix}_FixN_def_z0', we're asking folks to use the default z0 values from their host model.
Hopefully all of this makes sense -- please let me know if not, and again, apologies for the confusion.
Thanks,
Tim

**Reply by apossner** [reply]
Thanks!
